pynicotine/gtkgui/mainwindow.py

```diff
@@ -551,10 +552,13 @@ def create_hamburger_menu(self):
 
         menu = PopupMenu(self.application)
         self.add_connection_section(menu)
-        self.add_configure_shares_section(menu)
-        self.add_browse_shares_section(menu)
-
-        menu.add_items((">" + _("_Help"), self.create_help_menu()))
+        menu.add_items(
+            ("#" + _("_Rescan Shares"), "app.rescan-shares"),
+            (">" + _("_Browse Shares"), self.create_browse_shares_menu()),
+            ("#" + _("Con_figure Shares"), "app.configure-shares"),
```

Comment:
"Con_figure Shares" -> "_Configure Shares"

New unique translatable string not required as there is no conflict within this menu.

Or consider "Configure _Shares"

Reply:
It conflicted with the "Connect" item.

Your suggestion is better.